colors.go

```diff
@@ -225,7 +231,13 @@ func (sm styleMap) parseGNU(env string) {
 			key = filepath.Clean(key)
 		}
 
-		sm[key] = applyAnsiCodes(val, tcell.StyleDefault)
+		if key == "ln" && val == "target" {
+			sm.useLinkTarget = true
+			log.Printf("using link target for styles")
```

Comment:
I think this is too trivial of a debug statement for logging, and can be removed. Normally logging is used for initializations and recording errors.